generator/internal/genclient/language/internal/rust/rust.go

```diff
@@ -231,11 +231,26 @@ func (c *Codec) HTTPPathFmt(m *genclient.PathInfo, state *genclient.APIState) st
 	return fmt
 }
 
+func (c *Codec) unwrapFieldPath(components []string, requestAccess string) (string, string) {
```

Comment:
nit: can you add a doc comment to this method to describe what is being returned.

Reply:
Good idea, done.